content/source/docs/enterprise-beta/api/policy-checks.html.md

```diff
@@ -0,0 +1,191 @@
+---
+layout: enterprise2
+page_title: "Policy Checks - API Docs - Terraform Enterprise Beta"
+sidebar_current: "docs-enterprise2-api-policy-checks"
+---
+
+# Policy Checks API
+
+-> **Note**: These API endpoints are in beta and are subject to change.
+
+## List policy checks
+
+This endpoint lists the policy checks in a run
+
+| Method | Path           |
+| :----- | :------------- |
+| GET | /runs/:run_id/policy-checks |
+
+### Parameters
+
+- `run_id` (`string: <required>`) - specifies the run ID for which to list policy checks
+
+### Sample Request
+
+```shell
+curl \
+  --header "Authorization: Bearer $ATLAS_TOKEN" \
+  --header "Content-Type: application/vnd.api+json" \
+  https://atlas.hashicorp.com/api/v2/runs/run-CZcmD7eagjhyXavN/policy-checks
+```
+
+### Sample Response
+
+```json
+{
+  "data": [
+    {
+      "id": "polchk-9VYRc9bpfJEsnwum",
+      "type": "policy-checks",
+      "attributes": {
+        "result": {
+          "result": false,
+          "passed": 0,
+          "total-failed": 1,
+          "hard-failed": 0,
+          "soft-failed": 1,
+          "advisory-failed": 0,
+          "duration-ms": 0,
+          "sentinel": {
```

Comment:
It might be worthwhile to mention some place that this field's structure may change without notice. This field contains raw Sentinel data, so any change in Sentinel's core itself can change the format of this field.

Reply:
Each of the APIs has a note on top saying that he API is in beta and subject to change. Until we take the beta tag off the API I think this should be sufficient. I'll start a thread in Slack to discuss how these changes will impact our API versioning strategy.

Reply:
https://app.asana.com/0/434292910648127/493393102787398